src/main/java/com/thinkful/noteful/auth/WebSecurityConfig.java

```diff
@@ -0,0 +1,65 @@
+package com.thinkful.noteful.auth;
+
+import static com.thinkful.noteful.auth.SecurityConstants.SIGN_UP_URL;
+
+import com.thinkful.noteful.users.UserRepository;
```

Comment:
`import com.thinkful.noteful.users.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;` - we can remove both of these too!

Reply:
Done